pbe/breakupCoalescenceTables.py

```diff
@@ -0,0 +1,51 @@
+import fluid
+import numpy as np
+import matplotlib.pyplot as plt
+
+path = 'validationData/tables/1/'
+f = open(path + 'parameters', 'w')
+f_br = open(path + 'breakup', 'w')
+f_c = open(path + 'coalescence', 'w')
+
+F = fluid.fluid('coulaloglou', 0, 0)
+F.C1 = 1.52e-02
+F.C2 = 6.78e-02
+F.C3 = 1.06e-12
+F.C4 = 5.13e13
```

Comment:
I'd prefer to have them in at least two separate variables since they appear as constants in two different functions. 
Is there any advantage of having them in length two vector or dictionaries over separate variables when you don't loop over them anywhere?